tests/test_commands.py

```diff
@@ -62,9 +62,64 @@ def mock_bigchaindb_backup_config(monkeypatch):
     monkeypatch.setattr('bigchaindb._config', config)
 
 
+def test_make_sure_we_dont_remove_any_command():
+    # thanks to: http://stackoverflow.com/a/18161115/597097
+    from bigchaindb.commands.bigchain import utils
+    from bigchaindb.commands.bigchain import create_parser
+
+    parser = create_parser()
+
+    with pytest.raises(SystemExit):
+        utils.start(parser, [], {})
```

Comment:
It seems to me that this:

```python
with pytest.raises(SystemExit):
    utils.start(parser, [], {})
```

could be in its own separate test, e.g., something like `test_utils_start_with_no_args_command()`.